README.md

```diff
@@ -1,16 +1,17 @@
 [![CircleCI](https://circleci.com/gh/giantswarm/athena.svg?style=shield&circle-token=e86584296950c3be820856e802a37336c2c7d540)](https://circleci.com/gh/giantswarm/athena)
 [![Docker Repository on Quay](https://quay.io/repository/giantswarm/athena/status?token=789fdfa0-2068-473d-9ab6-7bde4aaf46dc "Docker Repository on Quay")](https://quay.io/repository/giantswarm/athena)
 
-# athena
+# Athena
 
-athena is a service that knows everything about your cluster. Its purpose is to provide metadata to clients, so they could easily establish a connection with the Kubernetes API, and identify the cluster that they're talking to.
+Athena is a service that knows some useful about your cluster. Its purpose is to provide some none-sensitive data (e. g. the CA certificate of the Kubernetes API, the cluster identifier, the cloud provider) to public clients, so they could easily establish a connection with the Kubernetes API, and identify the cluster that they're talking to.
```

Comment:
```suggestion
Athena is a service that knows some useful things about your cluster. Its purpose is to provide some none-sensitive data (e. g. the CA certificate of the Kubernetes API, the cluster identifier, the cloud provider) to public clients, so they could easily establish a connection with the Kubernetes API, and identify the cluster that they're talking to.
```
Not sure if "things" is the best choice, but I think we could do with a noun here.